packages/main/src/Label.js

```diff
@@ -112,9 +112,8 @@ class Label extends UI5Element {
 		return labelCss;
 	}
 
-	onclick() {
+	_onClick() {
```

Comment:
We did not propose a convention until now. We discussed it briefly and decided to go for "_onclick"
or in other words "_on" + "domevent_name"

The commit message is not 100% accurate. The events have been always private, the way we subscribe is different and the message should always start with a verb (imperative form).

Instead of
refactor(ui5-label): event handlers are now private
write this:
refactor(ui5-label): subscribe event handlers within HBS template

Reply:
Thank you for the feedback. All comments applied